app/views/posts/_post_cards.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class='user-section-card'>
+    <div class="comment-card">
+        <div class="user-card__name">
+            <h3> <%= post.title %> </h3>
+            <p class="user-bio"><%= post.text %></p>
+        </div>
+        <div class="comment__like">
+            <p class="number-posts">Comments:<%= post.comments_counter %> </p>  
+            <p class="number-posts">Likes:<%= post.likes_counter %> </p>  
+        </div>
+        <%= link_to "See Post", user_post_path(post.user, post), class: "btn" %>
+    </div>
+</section>
```

Comment:
- [ ] As requested by the previous reviewer, you should use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page, not `app/views/posts/show.html.erb` page.
The post-show page should display post with **all comments** listed as shown on the wireframe.